frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -63,18 +63,16 @@ export const RobotPage = () => {
                                 {selectedRobot.status !== RobotStatus.Offline && (
                                     <>
                                         <BatteryStatusDisplay itemSize={48} batteryLevel={selectedRobot.batteryLevel} />
-                                        {selectedRobot.model.upperPressureWarningThreshold && (
```

Comment:
Could we not just replace selectedRobot.model.upperPressureWarningThreshold here with selectedRobot.pressureLevel? Since that would remove the need for the first if statement in the PressureStatusDisplay component. Then we could avoid rendering the PressureStatusDisplay component, and it would be visible in the robotPage, when the component would not display anything